app/database/database.py

```diff
@@ -1,9 +1,14 @@
+import os
+
+from dotenv import load_dotenv
 from sqlalchemy import create_engine
 from sqlalchemy.ext.declarative import declarative_base
 from sqlalchemy.orm import sessionmaker
 
 
-SQLALCHEMY_DATABASE_URL = "sqlite:///./dev.db"
+load_dotenv()
+
+SQLALCHEMY_DATABASE_URL = os.getenv("DB_URL")
```

Comment:
The formal name is "database connection string".

Maybe we should change the name to `DATABASE_CONNECTION_STRING`?